src/bacnetmstp/CMakeLists.txt

```diff
@@ -1,11 +1,11 @@
-set (libname "bacnetmstp")
-set (libdescription "Driver module for BACnet MS/TP devices")
-set (module_src ${libname}.cxx device-client.c bacnetutil.cxx)
-set (module_hpp ${libname}.hpp bacnetutil.hpp)
-
 if (BACNET_FOUND)
+  set (libname "bacnetmstp")
+  set (libdescription "Driver module for BACnet MS/TP devices")
+  set (module_src ${libname}.cxx device-client.c bacnetutil.cxx)
+  set (module_hpp ${libname}.hpp bacnetutil.hpp device.h)
+
   set (reqlibname "libbacnet")
-  include_directories(${BACNET_INCLUDE_DIRS})
-  upm_module_init()
-  upm_target_link_libraries(${libname} ${BACNET_LIBRARIES} ${CMAKE_THREAD_LIBS_INIT})
+  upm_module_init(${BACNET_LIBRARIES} ${CMAKE_THREAD_LIBS_INIT})
+  # Add the bacnet include dirs to this target
+  target_include_directories(${libname} PUBLIC ${BACNET_INCLUDE_DIRS})
```

Comment:
Just a question/observation - We need this (target_include_directories) in this case because libbacnet is external, and not in the standard /usr/include/ correct?  For inter-upm dependencies, we only need specify them in upm_module_init() or REQUIRES (in the new-style) right?  All includes/lib dependencies are automatically added?

Just want to clarify my understanding...